events/beam-robotics.html

```diff
@@ -0,0 +1,154 @@
+    </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="event-section event-section-reverse">
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/codingrobot.svg" alt="a girl coding her robot"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 4: Coding our Robot</a> 
+      </h2>
+      <p>
+        Robots. Coding. Finally time to code robots?? In this lesson, you will learn how to make your robot move at your command!  
+      </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="event-section"> 
+    <div class="grid-col">
+      <img class="img-responsive" src="{{site.baseurl}}/img/beam-robotics/dancingrobot.svg" alt="a cartoon robot assembling itself"/>
+    </div>
+    <div>
+      <h2 class="title text-1.5x">
+        <a class="tla-link" href="#" target="_blank" rel="noopener noreferrer">Week 5: Making our Robot Dance</a>  
+      </h2>
+      <p>
+        In this final lesson, we will learn how to make our robot dance! Want to learn how to make your robot do more cool things like how to follow a line? See below for more robot ideas and materials!
+    </p>
+      <div class="button-container">
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fab fa-youtube"></span> Video</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-book"></span> Student Guide</a>
+        <a class="button button-dark" href="https://teachla.uclaacm.com/" target="_blank" rel="noopener noreferrer"> <span class="fas fa-chalkboard-teacher"></span> Instructor Guide</a>
+      </div>
```

Comment:
since we don't have these available yet, can we leave these out? ill bop you when we have the info!